runbotics/runbotics-common/src/Expressions/index.ts

```diff
@@ -68,10 +68,28 @@ export class Expressions {
             };
 
             response.jexl = true;
-            response.result = templatedString.replace(jexlPattern, (expressionMatch, innerProperty) => {
-                const evaluatedProperty = Jexl.evalSync(innerProperty, jexlContext);
-                return evaluatedProperty;
-            })
+
+            let isPropertyArray = false;
```

Comment:
avoid using let. function programming/useState only

Reply:
I think the simplest is replacing:
if (isPropertyArray) {

with:
if(Array.isArray(JSON.parse(property)))

Reply:
In this case I need information about evaluatedProperty specifically from inside of replace function. Without JSON.stringify, replace will return an array parsed to String, which causes the bug.